samples/Gallery/Pages/TreeView/EditableTreeView.fs

```diff
@@ -0,0 +1,339 @@
+namespace Gallery
+
+open System.Collections.ObjectModel
+open System.Diagnostics
+open System.Runtime.CompilerServices
+open Avalonia.Controls
+open Avalonia.Input
+open Avalonia.Layout
+open Avalonia.VisualTree
+open Fabulous.Avalonia
+open Fabulous
+
+open type Fabulous.Avalonia.View
+
+module FocusAttributes =
+    /// Allows setting the Focus on an Avalonia.Input.InputElement
+    let Focus =
+        Attributes.defineBool "Focus" (fun oldValueOpt newValueOpt node ->
+            let target = node.Target :?> InputElement
+
+            let rec focusAndCleanUp x y =
+                target.Focus() |> ignore
+                target.AttachedToVisualTree.RemoveHandler(focusAndCleanUp) // to clean up
+
+            if newValueOpt.IsSome && newValueOpt.Value then
+                (* TODO setting the focus on an AutoCompleteBox is broken.
```

Comment:
Tried a similar scenario Xaml C# and  the Focus doesn't seem to work either. I guess this is a limitation.